apps/music/views/player/view.js

```diff
@@ -119,7 +119,7 @@ PlayerView.prototype.next = function() {
 
 PlayerView.prototype.share = function() {
   this.getPlaybackStatus().then((status) => {
-    this.fetch('/api/activities/share/' + status.filePath);
+    this.fetch('/api/activities/share/' + encodeURIComponent(status.filePath));
```

Comment:
I just had a thought about this. So, `this.fetch` comes from the "js/view.js" base `View` class. It either maps the call to a bridge.js method or uses `window.fetch` if Service Workers are enabled. We can probably just put the `encodeURIComponent()` call inside of the `fetch` method in the base `View` class instead of having to do this everywhere.